panel/panelpluginsmodel.cpp

```diff
@@ -246,9 +246,12 @@ QString PanelPluginsModel::findNewPluginSettingsGroup(const QString &pluginType)
     groups.sort();
 
     // Generate new section name
-    for (int i = 2; true; ++i)
-        if (!groups.contains(QStringLiteral("%1%2").arg(pluginType).arg(i)))
-            return QStringLiteral("%1%2").arg(pluginType).arg(i);
+    if (!groups.contains(QStringLiteral("%1").arg(pluginType)))
+        return QStringLiteral("%1").arg(pluginType);
+    else
+        for (int i = 2; true; ++i)
+            if (!groups.contains(QStringLiteral("%1%2").arg(pluginType).arg(i)))
```

Comment:
@palinek Well, you’re right, again :smile:  I wasn’t sure it would be “well received” to modify *working* code I haven’t written.  If that is ok with you, I’d like to extend your optimization to the first case as well, with the same `QString`, as there is no need to use more memory than warranted. Which would give something like:

```
248     // Generate new section name
249     QString pluginName = QStringLiteral("%1").arg(pluginType);
250     
251     if (!groups.contains(pluginName))
252         return pluginName;
253     else
254     {   
255         for (int i = 2; true; ++i)
256         {   
257             pluginName = QStringLiteral("%1%2").arg(pluginType).arg(i);
258             if (!groups.contains(pluginName))
259                 return pluginName;
260         }
261     }
262 }
```
Not used to this indent style but I believe it to be compliant with the project.